File: src/main/java/org/demoth/aworlds/server2/api/messaging/fromClient/CommandMessage.java
package org.demoth.aworlds.server2.api.messaging.fromClient;

import org.demoth.aworlds.server2.api.messaging.Message;

public class CommandMessage extends Message {
    public Message action;

    @Override
    public String toString() {
        return "CommandMessage{" +
                "action=" + action +
                '}';
    }
}

File: src/main/kotlin/org/demoth/aworlds/server2/api/messaging/fromClient/CommandMessage.kt
package org.demoth.aworlds.server2.api.messaging.fromClient

import org.demoth.aworlds.server2.api.messaging.Message

class CommandMessage : Message() {
    var action: Message? = null

    override fun toString(): String {
        return "CommandMessage{" +
                "action=" + action +
                '}'.toString()
    }
}
